rm/rm-client/src/main/java/org/ow2/proactive/resourcemanager/core/properties/PAResourceManagerProperties.java

```diff
@@ -180,6 +181,20 @@
     /** Frequency of node history removal (cron expression) */
     RM_HISTORY_REMOVAL_CRONPERIOD("pa.rm.history.removal.cronperiod", PropertyType.STRING, "*/10 * * * *"),
 
+    /**
+     * Defines the frequency of attempts to remove {@link NodeState#DOWN} or
+     * {@link NodeState#LOST} node (cron expression)
+     */
+    RM_UNAVAILABLE_NODES_REMOVAL_FREQUENCY("pa.rm.nodes.unavailable.removal.frequency", PropertyType.STRING, "*/30 * * * *"),
+
+    /**
+     * Defines the period, in minutes, after which a {@link NodeState#DOWN} or
+     * {@link NodeState#LOST} node is eligible to automatic removal. The
+     * default value is 24h (1440 minutes). If this property is not set, these
+     * nodes will never be removed automatically.
```

Comment:
as this property has a default value, it will always be set, to enforce this behavior, you should remove the default values in both properties

Reply:
Do you mean that I should remove both properties from the config/rm/settings.ini?

Reply:
no, but you should remove the default value in the properties, so the user can decide in the settings.ini file

Reply:
Ok I understand, I removed the default value of `RM_UNAVAILABLE_NODES_MAX_PERIOD` in `PAResourceManagerProperties`.